src/main/java/uk/gov/hmcts/ccd/data/user/CachedUserRepository.java

```diff
@@ -47,6 +47,11 @@ public IdamUser getUser() {
         return userRepository.getUser();
     }
```

Comment:
May we  you can add a test in CachedUserRepositoryTest.  Optional It is not mandatory if sonar already passed. :)

Reply:
I don't think we need the test here as it is trying to get from DefaultUserRepository always